101-networkroutes/azuredeploy.json

```diff
@@ -0,0 +1,90 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "namePrefix": {
+            "type": "string",
+            "metadata": { 
+                "description": "Name prefix that will be applied to all resources in this template." 
+            } 
+        }
+    },
+    "variables": {
+        "routeTable1Name": "[concat(parameters('namePrefix'), 'RouteTable1')]",
+        "routeTable2Name": "[concat(parameters('namePrefix'), 'RouteTable2')]",
+        "virtualNetworkName": "[concat(parameters('namePrefix'), 'VNet')]"
+    },
+    "resources": [
+		{
+			"apiVersion": "2015-06-15",
```

Comment:
Would probably be a good idea to start making apiVersion a variable to make it easy to move to a newer api version